open-metadata-resources/open-metadata-deployment/docker/ranger/Dockerfile

```diff
@@ -1,62 +1,26 @@
-# SPDX-License-Identifier: CC-BY-4.0 
-# Copyright Contributors to the Egeria project. 
-# ---
-# Docker configuration file for Ranger
-
-# Let's use UBUNTU
-FROM            ubuntu:18.04
-
-# Image metadata following http://label-schema.org/rc1/
```

Comment:
I notice this change has dropped the metadata labels.
One could discuss how relevant they are, though we should probably have them in all or images, or none
Some tips at https://medium.com/@chamilad/lets-make-your-docker-image-better-than-90-of-existing-ones-8b1e5de950d
We could perhaps replace the version with a simple build date, as that is easy to automate.
Probably the best approach is to simply open up a new issue to correct the tags, and allow this change to go through.

Reply:
Opened https://github.com/odpi/egeria/issues/1048 to follow up